pkg/operator/tektonresults.go

```diff
@@ -98,7 +98,7 @@ func VerifyResultsRecords(resourceType string) {
 		if err != nil {
 			testsuit.T.Errorf("Error decoding base64 data: %v", err)
 		}
-		fmt.Printf(string(decodedResultsLogs))
+		log.Printf("%s", decodedResultsLogs)
```

Comment:
`log.Print` should work, then we do not need the format verb `%s`

Reply:
cannot use decodedResultsLogs (variable of type []byte) as string value in argument to log.Printf) (typecheck)
we get this error after removal of %s.

Reply:
will recheck again

Reply:
My Bad when I read the review. Changed it to `log.Print`